docs/fides/docs/development/testing_environment.md

```diff
@@ -1,29 +1,43 @@
 # Testing Environment
 
-To facilitate thorough manual testing of the application, there is a comprehensive testing environment that can be set up via a single `nox` command.
+## Quickstart
+1. Use `nox -s "fides_env(test)"` to launch the test environment
+2. Read the terminal output for details
+3. Customize Fides ENV variables by editing `.env`
 
-## Configuration
+## Overview
+
+To facilitate thorough manual testing of the application, there is a comprehensive testing environment that can be set up via a single `nox` command: `nox -s "fides_env(test)"`.
 
-The environment will configure the `fides` server and CLI using the TOML configuration set in `src/fides/data/test_env/fides.test_env.toml`. To test out other configurations, you can edit this file and reload the test env; however, don't commit these changes unless you are sure that the default configuration for testing should change for everyone!
+This test environment includes:
+* Fides Server
+* Fides Admin UI
+* Fides Postgres Database & Redis Cache
+* Sample "Cookie House" Application
+* Test Postgres Database
+* Test Redis Database
+* Sample Resources
+* Sample Connectors
+* etc.
 
-## Secrets Management
+This test environment is exactly the same environment that users can launch themselves using `fides deploy up`, and you can find all the configuration and settings in `src/fides/data/sample_project`.
+
+## Configuration
 
-The environment will work "out of the box", but can also be configured with secrets needed to configure other features like S3 storage, Mailgun notifications, etc. To configure this, you'll need to create the `.env` file, place it at the root of the repository directory, and provide some secrets. There is an `example.env` file you can reference to see what secrets are supported.
+There are two ways to configure the `fides` server and CLI:
+1. Editing the ENV file in the project root: `.env`
+2. Editing the TOML file in the sample project files: `src/fides/data/sample_project/fides.toml`
 
-This `.env` file is ignored by git and therefore safe to keep in your local repo during development.
+The `.env` file is safest to add secrets and local customizations, since it is `.gitignore`'d and will not be accidentally committed to version control.
```

Comment:
Assuming someone doesn't accidentally update the `.gitignore` and forget about it 🙃 good thing I haven't done that before hah